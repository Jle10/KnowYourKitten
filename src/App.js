function App() {
  return (
    <h1>
      Know your kitten
    </h1>
  );
}

export default App;
