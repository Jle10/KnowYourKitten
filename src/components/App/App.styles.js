import styled from "styled-components";
import background from "../../assets/background.jpg";

export const MainContainer = styled.main`
  align-items: center;
  background-image: url(${ background });
  background-repeat: unset;
  background-size: cover;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  min-height: 100vh;
  width: 100%;
`;

export const Overlay = styled.div`
  background-color: rgba(255, 255, 255, 0.8);
  height: 100%;
  left: 0;
  position: fixed;
  top: 0;
  width: 100%;
`;