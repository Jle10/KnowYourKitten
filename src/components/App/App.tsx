import Title from '../Title/Title';
import Grid from '../Grid/Grid';
import Button from '../Button/Button';

import { MainContainer, Overlay } from './App.styles';

function App() {
    return (
        <MainContainer data-testid="main-container">
            <Title />
            <Grid />
            <Button />
            <Overlay />
        </MainContainer>
    );
}

export default App;