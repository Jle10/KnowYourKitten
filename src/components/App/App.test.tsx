import { render, screen } from '@testing-library/react';
import App from './App';

test('renders and tests the app', () => {
    render(<App />);
    const mainContainer = screen.getByTestId('main-container')
    expect(mainContainer).toBeInTheDocument();
    expect(mainContainer.tagName).toBe('MAIN');
});
