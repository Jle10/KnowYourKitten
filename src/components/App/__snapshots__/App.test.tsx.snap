// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and tests the app 1`] = `Array []`;
